packages/@ngtools/webpack/src/type_checker_messages.ts

```diff
@@ -0,0 +1,30 @@
+// @ignoreDep typescript
```

Comment:
Can you move the `TypeChecker` class here (along with most things really), rename this file `type_checker.ts`, while leaving `type_checker_worker.ts` with only the messaging parsing code please?

It's likely enough that the type checker class might be needed individually from the worker in the future.

Reply:
Ah, good idea. Will do sir!